### PR TITLE
Testcontainers via test-resources version catalog

### PR DESCRIPTION
**Pull request by wetted**

### CODE REVIEW

File: benchmarks/benchmark-spring-data-mongodb/build.gradle [hunk 1]
```diff
@@ -6,6 +6,5 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-data-mongodb'
 	implementation mnMongo.mongo.core
 	implementation mnMongo.mongo.driver
-	testImplementation platform(libs.testcontainers.bom)
-	testImplementation libs.testcontainers.mongodb
+    implementation(mnTestResources.testcontainers.mongodb)
```
Comment (sdelamo) [inline benchmarks/benchmark-spring-data-mongodb/build.gradle:9]:
```suggestion
    testImplementation(mnTestResources.testcontainers.mongodb)
```

File: data-connection-jdbc/build.gradle [hunk 2]
```diff
@@ -21,13 +21,11 @@ dependencies {
...
     testImplementation mnReactor.micronaut.reactor
 
-    testImplementation platform(libs.testcontainers.bom)
-    testImplementation libs.testcontainers.spock
-    testImplementation libs.testcontainers.mysql
-    testImplementation libs.testcontainers.mariadb
-    testImplementation libs.testcontainers.postgresql
-    testImplementation libs.testcontainers.mssqlserver
-    testImplementation libs.testcontainers.oracle
+    implementation(mnTestResources.testcontainers.mysql)
```
Comment (sdelamo) [inline data-connection-jdbc/build.gradle:24]:
```suggestion
    testImplementation(mnTestResources.testcontainers.mysql)
```

File: data-connection-jdbc/build.gradle [hunk 3]
```diff
@@ -21,13 +21,11 @@ dependencies {
...
+    implementation(mnTestResources.testcontainers.mariadb)
```
Comment (sdelamo) [inline data-connection-jdbc/build.gradle:25]:
```suggestion
    testImplementation(mnTestResources.testcontainers.mariadb)
```

File: data-connection-jdbc/build.gradle [hunk 4]
```diff
@@ -21,13 +21,11 @@ dependencies {
...
+    implementation(mnTestResources.testcontainers.postgres)
```
Comment (sdelamo) [inline data-connection-jdbc/build.gradle:26]:
```suggestion
    testImplementation(mnTestResources.testcontainers.postgres)
```

File: data-connection-jdbc/build.gradle [hunk 5]
```diff
@@ -21,13 +21,11 @@ dependencies {
...
+    implementation(mnTestResources.testcontainers.mssql)
```
Comment (sdelamo) [inline data-connection-jdbc/build.gradle:27]:
```suggestion
    testImplementation(mnTestResources.testcontainers.mssql)
```

File: data-connection-jdbc/build.gradle [hunk 6]
```diff
@@ -21,13 +21,11 @@ dependencies {
...
+    implementation(mnTestResources.testcontainers.oracle.xe)
```
Comment (sdelamo) [inline data-connection-jdbc/build.gradle:28]:
```suggestion
    testImplementation(mnTestResources.testcontainers.oracle.xe)
```

File: data-jdbc/build.gradle [hunk 7]
```diff
@@ -37,13 +37,11 @@ dependencies {
...
     testImplementation mnValidation.micronaut.validation.processor
 
-    testImplementation platform(libs.testcontainers.bom)
-    testImplementation libs.testcontainers.spock
-    testImplementation libs.testcontainers.mysql
-    testImplementation libs.testcontainers.mariadb
-    testImplementation libs.testcontainers.postgresql
-    testImplementation libs.testcontainers.mssqlserver
-    testImplementation libs.testcontainers.oracle
+    implementation(mnTestResources.testcontainers.mysql)
```
Comment (sdelamo) [inline data-jdbc/build.gradle:40]:
```suggestion
    testImplementation(mnTestResources.testcontainers.mysql)
```

File: data-jdbc/build.gradle [hunk 8]
```diff
@@ -37,13 +37,11 @@ dependencies {
...
+    implementation(mnTestResources.testcontainers.mariadb)
```
Comment (sdelamo) [inline data-jdbc/build.gradle:41]:
```suggestion
    testImplementation(mnTestResources.testcontainers.mariadb)
```

File: data-jdbc/build.gradle [hunk 9]
```diff
@@ -37,13 +37,11 @@ dependencies {
...
+    implementation(mnTestResources.testcontainers.postgres)
+    implementation(mnTestResources.testcontainers.mssql)
```
Comment (sdelamo) [inline data-jdbc/build.gradle:43]:
```suggestion
    testImplementation(mnTestResources.testcontainers.mssql)
```

File: data-r2dbc/build.gradle [hunk 10]
```diff
@@ -43,9 +43,7 @@ dependencies {
     testImplementation mn.micronaut.http.server.netty
     testImplementation mn.micronaut.http.client
 
-    testImplementation platform(libs.testcontainers.bom)
-
-    testImplementation libs.testcontainers.r2dbc
+    implementation(mnTestResources.testcontainers.r2dbc)
```
Comment (sdelamo) [inline data-r2dbc/build.gradle:46]:
```suggestion
    testImplementation(mnTestResources.testcontainers.r2dbc)
```

**Review by sdelamo — CHANGES_REQUESTED** [review]
Please, it is important not to change the Gradle configuration. These dependencies should be `testImplementation` not `implementation`.

**Comment by wetted** [comment]
> Please, it is important not to change the Gradle configuration. These dependencies should be `testImplementation` not `implementation`.

Thanks for catching that. It was quite unintentional.

**Review by sdelamo — APPROVED** [review]

**Pull request merged**